webserver/src/test/java/org/jboss/test/arquillian/ce/webserver/WebserverTestBase.java

```diff
@@ -56,15 +79,61 @@ protected static WebArchive getDeploymentInternal() throws Exception {
     }
 
     /*
+    * Allow httpClient to use untrusted connections
+    * The code below was tested with httpclient 4.3.6-redhat-1
+    * code example from; http://literatejava.com/networks/ignore-ssl-certificate-errors-apache-httpclient-4-4/
+    */
+    public static HttpClient acceptUntrustedConnClient() throws Exception {
```

Comment:
Who adds this dependency -- HttpClient -- to deployment?
As this is gonna fail if it's not there.

Or even better, make it lazy, so we don't have to add it to deployment(s).